tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -820,19 +770,990 @@ describe(__filename, () => {
       // individual tests above test separate pieces of logic.
       it('returns the expected URL for an add-on', () => {
         const addonId = 123;
-        const addon = createInternalAddonWithLang({
+        const internalAddon = createInternalAddonWithLang({
           ...fakeAddon,
```

Comment:
This should now use `addon` instead of `fakeAddon`. In cases like this you could call `addon.id = addonId;` and `addon.guid = guid` first, and then pass `addon` into the call, avoiding the destructuring.